strawberry_django/fields/types.py

```diff
@@ -60,6 +61,13 @@ class ManyToManyInput:
     set: Optional[List[strawberry.ID]] = UNSET
 
 
+Point = strawberry.scalar(
+    NewType("Point", (Optional(float), Optional(float), Optional(float))),
```

Comment:
I don't think this is correct.

1) The typing in this case should be `Tuple[Optional(float), Optional(float), Optional(float)]`

2) The arguments are not actually optional, trying to pass `None` to any of those is an error

3) The third argument is not mandatory. You can create a point with 2 or 3 floats (you just can't use 1 or more than 3)

The best case in here would be to use `Union[Tuple[float, float], Tuple[float, float, float]]`, but `NewType` doesn't accept it. The correct way would probably be `Tuple[float, ...]`